src/pyrtma/context.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import copy
+
+from typing import Dict, Any
+from .message_base import MessageBase
+
+
+def _create_context() -> Dict[str, Dict[str, Any]]:
+    ctx: Dict[str, Dict[str, Any]] = dict(
```

Comment:
Maybe make a [TypedDict ](https://docs.python.org/3/library/typing.html#typing.TypedDict)class for contexts?